nix/gce-images.nix

```diff
@@ -0,0 +1,7 @@
+let self = {
+  "18.03" = "nixos-1803";
+  "18.09" = "nixos-1809";
+
+  latest = self."18.09";
+  project = "predictix-operations";
```

Comment:
This should be updated to a "trusted" upstream like nixos/nix-community.

Reply:
Definitely, added the steps to prepare the images in the description :

_Building an image from source_
```
$ gcloud compute images create nixos-18091228a4c4cbb613c-x86-64-linux  \
              --source-uri gs://nixos-cloud-images/nixos-image-18.09.1228.a4c4cbb613c-x86_64-linux.raw.tar.gz \
              --family=nixos-1809
```
_Making the image public_
```
$ gcloud compute images  add-iam-policy-binding nixos-18091228a4c4cbb613c-x86-64-linux \
             --member='allAuthenticatedUsers'                                                                              \
             --role='roles/compute.imageUser'  
```
@AmineChikhaoui should be familiar with these operations

Reply:
I think we should update the `create-gce.sh` script in nixpkgs to add those steps. Thanks for researching this !
@adisbladis yeah we'll have to host this under a NixOS account. I'm looking into that. Also we're missing a couple of releases in GCP images, so will start first with 20.03 and see if it works.